README.md

```diff
@@ -108,6 +109,11 @@ Some ideas for getting started:
 For more details see the [`CONTRIBUTING.md`][contributing]
 
 
+## Visual Overview
```

Comment:
Can we make this a subsection of [Explanations](https://github.com/spacchetti/spago#explanations) and call it something like "What happens when you do `spago build`"?

Reply:
I kept the 'visual overview' prefix because I think it's good to emphasize that this is an image, not more text.

I think the image should appear sooner in the ReadMe. Still, I've made the changes you've requested.

Reply:
@JordanMartinez thanks! I actually agree with you and I think that there are other sections that need shuffling too. I'll open a PR in the next days after I think a bit more about this

Reply:
Sounds goo!